Resources/Prototypes/_Funkystation/Reagents/exotic.yml

```diff
@@ -126,11 +126,32 @@
   physicalDesc: reagent-physical-desc-fluffy
   flavor: medicine
   color: "#ffaa00"
+  metabolisms:
+    Poison:
+      effects:
+      - !type:CreateEntityReactionEffect # DeltaV - was polymorph, just have a chance to spawn a corgi instead
+        entity: MobCorgiPuppy
+        probability: 0.25
+        conditions:
+        - !type:OrganType
+          type: Animal
+          shouldHave: false
+        - !type:ReagentThreshold
+          min: 25 # DeltaV - need more corgium for that to happen
+
+- type: reagent
+  id: Muigroc
+  name: reagent-name-muigroc
+  group: Exotic
+  desc: reagent-desc-muigroc
+  physicalDesc: reagent-physical-desc-fluffy
+  flavor: medicine
+  color: "#ffaa00"
   metabolisms:
     Poison:
       effects:
       - !type:Polymorph
-        prototype: CorgiMorph # PERMANENT. There's a reason this chem's an exotic chem, not a low-mid level chem. Exotic's primary toxin.
+        prototype: CorgiMorphTemp # DeltaV - sorry perma corgi is too evil
```

Comment:
if someone is going to waste tc on the stupid valid chemical they should get some antag utility out of it, you could grind up lizard plushies instead if you just want a temp polymorph

Reply:
perma polymorph means polymorphing the nuke disk holder results in that getting sent to limbo and not respawning which is 

not good

Reply:
made the polymorph spicier but not permanent

Reply:
you can still make it permanent just drop the inventory

Reply:
5 minutes is still a very potent incapacitation

Reply:
"incapacitation" except it still has the gamers name and you instantly get outed as a traitor

Reply:
like if im going to inject/foam you with 26+ units of something (28 or 29 if hypoing it all depending on ping) why would i waste all this time and vestine making this instead of just using mutagen and calling it a day